content/future.html

```diff
@@ -2,104 +2,299 @@ <h1 id="future"> Future </h1>
-  many of the bogus technical concepts. I will, however, remember the late
-  nights in the SE lab with friends, fooling around while doing a last minute
-  assignment."
-  <br /><br />
-  Try out as many different co-ops as possible. And don't Cali or continuous.
-  It's tough to get your first job, but don't give up. And if you get a co-op at
-  a big company, it does not mean that you're at the top of the world.
-  <br /><br />
-  And most importantly, university goes by in a flash, so enjoy it!
-  <br /><br /><br />
-  - University of Waterloo Software Engineering Class of 2018
+    The best drink you can have is Burgundy's at Wilfs. I will
+    be coming back for those
+    <br></br>
+    If you want to be super organized when packing and moving
+    around, store photos of various layers of your packed bags.
+    Then, when you need to go on a similar trip (whether it's
+    2 weeks in a tropical location or moving to a new city for
+    co-op), you know exactly what to pack and where, and it
+    basically requires no time at all to get ready!
+    <br></br>
+    Take the same courses as the people you live with.
+    <br></br>
+    If you don't collect air miles you're missing out on
+    $10-$50 per flight.
+    <br></br>
+    Talk less, listen more.
+    <br></br>
+    Surround yourself with successful people who will naturally
+    push you to do your best.
+    <br></br>
+    Be empathetic.
   </p>
-</p>
+
+  <div class="vb"></div>
+  <span class="big">
+    "What is one piece of advice you would give to your
+    first-year self?"
+  </span>
+  <p class="quote">
+    Have confidence. You deserve to be here.
+    <br></br>
+    Overworking yourself never works! You reach a point
+    were putting more work doesn't yield the cost benefit
+    necessary, instead you will feel much better and
+    perform as well or better by being involved in something
+    you enjoy.
+    <br></br>
+    Don't take everything so seriously. Take the time to
+    learn even if it means lower marks, and take some time
+    to explore projects, hobbies and have fun!
+    <br></br>
+    I don't think my first-year self would have taken my
+    advice.
+    <br></br>
+    Spend more time having fun. Always keep well being and
+    quality of life in mind.
+    <br></br>
+    Don't limit your co-op applications to one place - especially
+    in first year. It's better to apply to a variety of jobs
+    in a variety of locations -- use that as an excuse to
+    lightly travel when on coop.
+    <br></br>
+    Stop sending cringey snaps when drunk.
+    <br></br>
+    Buy doge coin.
+    <br></br>
+    People are less intimidating than you think.
+    <br></br>
+    Take more risks throughout university
+    <br></br>
+    Be yourself.
+  </p>
+
+  <div class="vb"></div>
+  <span class="big">
+    "What was the best lesson you learned in your undergrad?"
+  </span>
+  <p class="quote">
+    Compassion
```

Comment:
nit (for the end period)
```suggestion
    Compassion.
```